MockerTests/MockerTests.swift

```diff
@@ -290,7 +290,7 @@ final class MockerTests: XCTestCase {
         let simpleURL = URL(string: "https://host.com/api/resource")
         let mock = Mock(url: composedURL, dataType: .json, statusCode: 200, data: [.get: MockedData.exampleJSON.data])
         let urlRequest = URLRequest(url: simpleURL!)
-        XCTAssertEqual(composedURL.absoluteString, simpleURL!.absoluteString)
+        XCTAssertEqual(composedURL.absoluteString, simpleURL?.absoluteString)
         XCTAssert(mock == urlRequest)
```

Comment:
This one I didn't change; somehow it is comparing a `Mock` to a `URLRequest`... but `XCTAssertEqual` didn't seem to be able to figure out the generics, so I left it as-is.